datadog/threadstats/aws_lambda.py

```diff
@@ -61,6 +63,8 @@ def _close(cls):
                     _lambda_stats.flush(float("inf"))
 
     def __call__(self, *args, **kw):
+        warnings.warn(
+            "datadog_lambda_wrapper() is deprecated; use datadog-lambda package instead https://github.com/DataDog/datadog-lambda-layer-python", DeprecationWarning)
```

Comment:
"datadog_lambda_wrapper() is deprecated" can potentially cause confusion. For users, it just has been relocated, e.g., user still need to import `datadog_lambda_wrapper`, but from a different location. Perhaps `datadog_lambda_wrapper() is relocated to https://github.com/DataDog/datadog-lambda-layer-python`?

Reply:
BTW, is it better to add this warning in the `__init__` method instead?

Reply:
I think if it's in init it won't get caught in cloud watch logs, because it's outside the invocation.

Reply:
I updated the url to use githubs link shortener, otherwise the line trips up the linter for being too long.